src/status_im/chat/models.cljs

```diff
@@ -453,8 +453,10 @@
     :content             (i18n/label :t/clear-history-confirmation-content)
     :confirm-button-text (i18n/label :t/clear-history-action)
     :on-accept           #(do
```

Comment:
this one can be simpler, do is redundand and better to use fn for parent function and # inside

```clojure
(fn []
      (rf/dispatch [:bottom-sheet/hide
                              #(re-frame/dispatch [:chat.ui/clear-history chat-id false]))]))}})